### PR TITLE
Update default branch to main

### PR DESCRIPTION
**Pull request by KPrasch**

### CODE REVIEW

**Comment by derekpierre** [comment]
I'm unsure whether `latest` on readthedocs will pick up from `main` automatically, but it probably makes sense to explicitly specify the default branch to be `main` in the advanced settings - https://readthedocs.org/dashboard/nucypher/advanced/. wdyt?

**Comment by derekpierre** [comment]
Will the `README.md` file also need to be changed? https://github.com/nucypher/nucypher/blame/main/README.md#L8

File: docs/source/guides/contribution_guide.rst [hunk 1]
```diff
@@ -113,7 +113,7 @@ Pull Request Conflicts
 ----------------------
 
 As an effort to preserve authorship and a cohesive commit history, we prefer if proposed contributions
-are rebased over master (or appropriate branch) when a merge conflict arises,
+are rebased over `main` (or appropriate branch) when a merge conflict arises,
```
Comment (cygnusv) [inline docs/source/guides/contribution_guide.rst:116]:
```suggestion
are rebased over ``main`` (or appropriate branch) when a merge conflict arises,
```

**Comment by KPrasch** [comment]
@derekpierre - Yes - I think that will suffice.

File: docs/source/guides/contribution_guide.rst (hunk 1 above)
Comment (KPrasch) [reply docs/source/guides/contribution_guide.rst:116]: I do not see this diff locally :thinking:

**Comment by KPrasch** [comment]
> Will the README.md file also need to be changed?

Yes.

**Pull request merged**